index.html

```diff
@@ -266,5 +266,42 @@ <h3>Skills</h3>
         <div class="line"></div>
       </div>
     </section>
+
+    <section class="contact">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I&#39;d love to help
+        with it!
+      </p>
+
+      <form
+        action="https://formspree.io/f/xknaqwvy"
+        method="post"
+        class="contact-form"
+      >
+        <input
+          type="text"
+          class="name"
+          placeholder="name..."
+          class="placeholder"
+        />
+        <input
+          type="email"
+          class="email"
+          placeholder="email..."
+          class="placeholder"
+        />
+        <textarea
+          class="placeholder"
+          name="message"
+          id=""
+          cols="30"
+          rows="10"
+          placeholder="Write your message here..."
+        ></textarea>
+        <button type="submit" class="button">Get in touch</button>
```

Comment:
- [ ] Kindly implement `hover` for the button interaction as per the project requirements.

- ![image](https://user-images.githubusercontent.com/60083437/228476736-a3abbd68-6582-4b1f-a91d-04cd3feef745.png)